gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/HiveSource.java

```diff
@@ -230,6 +220,25 @@ private void createWorkunitsForPartitionedTable(HiveDataset hiveDataset,
     }
   }
 
+  @VisibleForTesting
+  public boolean shouldCreateWorkunitForPartition(Partition partition, long updateTime, LongWatermark lowWatermark) {
```

Comment:
Made the first change to use ```shouldCreateWorkunit``` instead of ```shouldCreateWorkUnitForPartition``` and ```shouldCreateWorkUnitForTable```. 
I did not move ```isOlderThanLookback``` into ```shouldCreateWorkunit```, I want to avoid reading the update time for partitions older than ```lookbackTime```